web_interface/blueprints/api_v3.py

```
@@ -1716,33 +1716,34 @@ def execute_system_action():
             if mode:
                 # For on-demand modes, we would need to integrate with the display controller
                 # For now, just start the display service
-                result = subprocess.run(['sudo', 'systemctl', 'start', 'ledmatrix'],
-                                     capture_output=True, text=True)
+                result = subprocess.run(['sudo', 'systemctl', 'start', 'ledmatrix.service'],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)